bin/tictactoe

```diff
@@ -1,3 +1,27 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+puts "What's updog?"
+
+input = ""
+
+until input == "1" || input == "0" || input == "2"
+  puts "Which mode do you want? (0, 1, or 2 Player)"
+  input = gets.strip
+end
+
+if input == "0"
+  game = Game.new(Computer.new("X"), Computer.new("O"))
+  game.play
+end
+
+if input == "1"
+  game = Game.new(Human.new("X"), Computer.new("O"))
+  game.play
+end
+
+if input == "2"
+  game = Game.new(Human.new("X"), Human.new("O"))
+  game.play
+end
```

Comment:
Prefer to encapsulate CLI logic into a class. For example, the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
       
require_relative '../config/environment'

TicTacToeCLI.new.call
```

And then you'd have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def initialize
  end

  def call
   # Your game logic
  end
end
```

The code in your bin would generally just go in `call`.